Source/Core/VideoBackends/OGL/main.cpp

```diff
@@ -140,7 +140,7 @@ static void InitBackendInfo()
 	g_Config.backend_info.bSupportsGeometryShaders = true;
 	g_Config.backend_info.bSupports3DVision = false;
 	g_Config.backend_info.bSupportsPostProcessing = true;
-	g_Config.backend_info.bSupportsPaletteConversion = false;
+	//g_Config.backend_info.bSupportsPaletteConversion = true; // is GPU dependent and must be set in renderer
```

Comment:
I just did it in the same way as in line 137 and 138. iirc this configs are cleared on startup, so do you want me to remove this lines?